src/guides/resolve-zil-domain.md

```diff
@@ -0,0 +1,203 @@
+---
+description: >-
+  This tutorial will teach you how to make a JSON RPC call to our Zilliqa smart
+  contract to get various records as well as owner address of a domain owner.
+---
+
+# Resolving .zil domain
+
+## Steps to resolve unstoppable domain
+
+In order to fetch any data from the Zilliqa blockchain you will need to do the following:
+
+1. Get a namehash of a domain
+2. Get resolver and owner addresses
+3. Get the contract sub-state related to your domain
+
+### Namehashing
+
+Namehashing is an algorithm that converts a domain name in a classical format \(like www.example.crypto\) to ERC-721 token id. This process is described  in more details over  [here](https://docs.unstoppabledomains.com/domain-registry-essentials/namehashing)
+
+```typescript
+import hash from 'hash.js';
```

Comment:
Is it this library? https://www.npmjs.com/package/hash.js/v/1.0.1

I'd probably consider using something else, which is more common in JS. Afaik, NodeJS has its own `crypto`. Or we could either use this: https://www.npmjs.com/package/js-sha256